@JRC/processArgs.m

```diff
@@ -153,7 +153,7 @@ function processArgs(obj)
                 obj.hCfg = hCfg_;
                 obj.res = res_;
 
-                obj.saveRes();
+                obj.saveRes(~obj.args{2});
```

Comment:
This is an ugly tweak, sorry, I should have written that cleaner, but some change is needed here to allow the JRClust user to trigger the `forceOverwrite` option in `saveRes` when calling 'import-ksort'.

Reply:
I'm importing spike sorting results from KiloSort in batches and I need a way to overwrite previous imports. Since I already have the `confirm_flag` option set to False (in `obj.args{2}`), the tweak here is to use it to set `forceOverwrite` to True.

Reply:
That's not a satisfying solution of course .... Sorry about that.

Reply:
No, it's perfectly fine. Just put a comment there :^)